### PR TITLE
KEYCLOAK-12490 Add missing details for jboss-logging event listener

### PR DESCRIPTION
**Pull request by stianst**

### CODE REVIEW

File: server_admin/topics/events/login.adoc [hunk 1]
```diff
@@ -79,6 +79,41 @@ This logging is very useful if you want to use a tool like Fail2Ban to detect if
 is trying to guess user passwords.  You can parse the log file for `LOGIN_ERROR` and pull out the IP Address. Then feed this information
 into Fail2Ban so that it can help prevent attacks.
 
+The Logging Event Listener logs events to the `org.keycloak.events` logger category. By default debug log events are not
+included in server logs. To enable these you can either configure the log level for the `org.keycloak.events` or change
+the log level used by the Logging Event Listener. This is done by editing the `standalone.xml`, `standalone-ha.xml`, or `domain.xml` files.
```
Comment (pnagesha) [inline server_admin/topics/events/login.adoc]: To include debug log events in server logs, edit the 'standalone.xml' file and change the log level used by the Logging Event listener. Alternately, you can configure the log level for  'org.keycloak.events'.

File: server_admin/topics/events/login.adoc [hunk 2]
```diff
@@ -79,6 +79,41 @@ This logging is very useful if you want to use a tool like Fail2Ban to detect if
...
+
+For example to change the log level add the following:
```
Comment (pnagesha) [inline server_admin/topics/events/login.adoc]: For example,

File: server_admin/topics/events/login.adoc [hunk 3]
```diff
@@ -79,6 +79,41 @@ This logging is very useful if you want to use a tool like Fail2Ban to detect if
...
+----
+<subsystem xmlns="urn:jboss:domain:logging:...">
+    ...
+    <logger category="org.keycloak.events">
+        <level name="DEBUG"/>
+    </logger>
+</subsystem>
+----
+
+Or to change the log level used by the Logging Event listener add the following:
```
Comment (pnagesha) [inline server_admin/topics/events/login.adoc]: remove Or. 
To change the log level used by the Logging event listener,

File: server_admin/topics/events/login.adoc [hunk 4]
```diff
@@ -79,6 +79,41 @@ This logging is very useful if you want to use a tool like Fail2Ban to detect if
...
+        <properties>
+          <property name="success-level" value="info"/>
+          <property name="error-level" value="error"/>
+        </properties>
+      </provider>
+    </spi>
+</subsystem>
+----
+
+Valid values for the log levels are `debug`, `info`, `warn`, `error` and `fatal`.
```
Comment (pnagesha) [inline server_admin/topics/events/login.adoc]: 'error', and 'fatal'.

File: server_admin/topics/events/login.adoc [hunk 5]
```diff
@@ -79,6 +79,41 @@ This logging is very useful if you want to use a tool like Fail2Ban to detect if
...
+
 The Email Event Listener sends an email to the user's account when an event occurs.
 The Email Event Listener only supports the following events at the moment:
```
Comment (pnagesha) [inline server_admin/topics/events/login.adoc]: Currently, the Email Event listener supports the following events:

File: server_admin/topics/events/login.adoc [hunk 6]
```diff
@@ -79,6 +79,41 @@ This logging is very useful if you want to use a tool like Fail2Ban to detect if
...
+          <property name="error-level" value="error"/>
+        </properties>
+      </provider>
+    </spi>
+</subsystem>
+----
+
+Valid values for the log levels are `debug`, `info`, `warn`, `error` and `fatal`.
+
 The Email Event Listener sends an email to the user's account when an event occurs.
```
Comment (pnagesha) [inline server_admin/topics/events/login.adoc:119]: listener; lower case l.

**Review by pnagesha — CHANGES_REQUESTED** [review]
Please make sure that the suggested changes do not interfere with the technical accuracy of the document.

File: server_admin/topics/events/login.adoc (hunk 1 above)
Comment (stianst) [reply server_admin/topics/events/login.adoc]: Updating

File: server_admin/topics/events/login.adoc (hunk 3 above)
Comment (stianst) [reply server_admin/topics/events/login.adoc]: Updating

File: server_admin/topics/events/login.adoc (hunk 4 above)
Comment (stianst) [reply server_admin/topics/events/login.adoc]: updating

File: server_admin/topics/events/login.adoc (hunk 6 above)
Comment (stianst) [reply server_admin/topics/events/login.adoc:119]: Email Event Listener is a name, so should be upper-case

File: server_admin/topics/events/login.adoc (hunk 5 above)
Comment (stianst) [reply server_admin/topics/events/login.adoc]: updating

**Pull request merged**